proto/bragi/dota2.proto

```diff
@@ -106,16 +106,36 @@ message Dota2CurrentMapStatePartialUpdate {
 
     // Replaces "barracks" property in Dota2Barracks.
     Dota2Barracks barracks = 5;
+
+    // Replaces "items" property in Dota2Items.
+    Dota2Items items = 6;
```

Comment:
How this would not work items and abilities are not part of Dota2CurrentMapState. I will explain it F2F.